### DISCUSSION TITLE
SKR PRO 1.2

### DISCUSSION THREAD

**Discussion by DenEric** [grblHAL/STM32F4xx#104] [Q&A]
I am trying to get grblhal installed on my SKR pro 1.2 with a STM32F407ZGT6 processor running at 168MHz and TMC2209 drivers. Can someone tell which files I need to update/create to be able to compile for this board

**Comment by terjeio** [comment]
Using the [Web Builder](http://svn.io-engineering.com:8080/) is the easiest? If you choose the bootloader version copy the generated binary to a SD card to flash it.

**Reply by DenEric** [reply]
Sorry, has no effect wat so ever. The board setup still refers to a 8MHz clock.

**Reply by terjeio** [reply]
So your board do not have an 8 MHz crystal as stipulated in the schematic?

**Reply by DenEric** [reply]
From the manual:
1) The main control uses the ARM-level Cortex-M4 series with a
32-bit dominant frequency of 168 MHz STM32F407ZGT6 chip,
performance is greatly improved;

**Reply by DenEric** [reply]
Looked a bit closer to the schematic and yes it has an 8MHz crystal on it. That then is not the problem. But when I insert the sd card there is no reaction, it is the marlin software that keeps on running.

**Reply by terjeio** [reply]
Did you reboot (power up) the controller after inserting the SD card? And did the filename change?
BTW I see that I have missed a symbol in the Web Builder configuration for the board, this may prevent grblHAL starting. I have added this now.

**Reply by DenEric** [reply]
I did reboot but the filename did not change, I'll have a test with the updated configuration

**Reply by DenEric** [reply]
Not working, it seems the card completely ignores the sd card.

**Reply by terjeio** [reply]
Has the bootloader been overwritten?
A way to test if it is working is to flash a new version of Marlin, does this work?

Do you have access to a ST-Link programmer?

**Reply by DenEric** [reply]
Found it, the sd card was formatted in 'ext4' and the board needs FAT32. Thank you for the assistance
